provider/maas/interfaces.go

```diff
@@ -191,12 +201,13 @@ func maasObjectNetworkInterfaces(maasObject *gomaasapi.MAASObject, subnetsMap ma
 			nicInfo.Address = network.NewAddressOnSpace(sub.Space, link.IPAddress)
 			spaceId, ok := subnetsMap[string(sub.CIDR)]
 			if !ok {
-				// The space we found is not recognised, no
-				// provider id available.
+				// The space we found is not recognised.
+				// No provider space info is available.
 				logger.Warningf("interface %q link %d has unrecognised space %q", iface.Name, link.ID, sub.Space)
 			} else {
 				nicInfo.Address.SpaceProviderId = spaceId
 				nicInfo.ProviderSpaceId = spaceId
+				logger.Debugf("interface %q link %d space %q", iface.Name, link.ID, sub.Space)
```

Comment:
is this useful to leave in? would it be better at trace?
It might be fine, I haven't watched "juju debug-log" to see if it is spammy. I just wanted to raise the question.

Reply:
I can't speak to day-to-day "in-theatre", but it was useful for me at bootstrap - not just the interfaces, but more-so the space resolutions. Adds 2 terse lines in that event.

Will defer to your judgement.

Reply:
Actually I _will_ change this to trace.

The change in the MAAS provider instance.Addresses has debug level logging of the detected interfaces in the same call-stack. Trace-level might still be useful at the generic level here.